orchestration/dagster_orchestration/pyproject.toml

```diff
@@ -6,26 +6,26 @@ authors = ["Your Name <you@example.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-dagster = "^0.10.5"
+argo-workflows = "^5.0.0"
+cached-property = "^1.5.2"
+dagster = "^0.11.0"
```

Comment:
Be sure to update the dagster version here as well https://github.com/DataBiosphere/hca-ingest/blob/master/ops/helmfiles/dagster/helmfile.lock#L8